src/main/python/janusgraph_python/core/datatypes/Circle.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Circle(object):
+    def __init__(self, longitude, latitude, radiusInKM):
+        """
+            The actual class representing Geographical Circle.
+
+        Args:
+            latitude (float): The latitude of circle's center.
+            longitude (float): The longitude of circle's center
+            radiusInKM (int): The radius of circle in KM.
+        """
+
+        self.latitude = float(latitude) * 1.0
+        self.longitude = float(longitude) * 1.0
+        self.radius = radiusInKM
+
+        status = self.__are_valid_coordinates()
+
+        if not status:
+            raise ValueError("Invalid Coordinates/Radius passed. "
+                             "Latitude needs to be b/w [-90, 90] and Longitude b/w [-180, 180] and Radius > 0")
+
+        pass
+
+    def getShape(self):
+        """
+            Return the Shape of Geometrical shape
+        Returns:
+            str
+        """
+        return "CIRCLE"
+
+    def toString(self):
```

Comment:
Isn't the usual way in Python to just override `__str__`? Where's the point in also having an additional `toString()` method?

Reply:
I added that initially as a mistake, planned to remove but forgot. Thanks for pointing that out @FlorianHockmann  :-)